src/Promitor.Core/Extensions/AzureEnvironmentExtensions.cs

```diff
@@ -14,5 +16,24 @@ public static string GetDisplayName(this AzureEnvironment azureCloud)
         {
             return azureCloud.Name.Replace("Azure", "").Replace("Cloud", "").Humanize(LetterCasing.Title);
         }
+
+        /// <summary>
+        ///    Get Azure environment information for custom cloud
+        /// </summary>
+        /// <param name="azureCloud">Microsoft Azure cloud</param>
+        /// <param name="endpoints">Endpoints for custom environment</param>
+        public static AzureEnvironment GetCustomAzureEnvironment(AzureEndpoints endpoints)
+        {
+            return new AzureEnvironment
```

Comment:
I presume they don't have a log analytics endpoint we can leverage instead?

Reply:
yes, AzureEnvironment doesn't have it